packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -51,8 +49,41 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
     [rtl, offset, position],
   )
 
-  React.useEffect(
+  const scheduleUpdate = React.useCallback(
+    () => {
+      if (popperRef.current) {
+        popperRef.current.scheduleUpdate()
+      }
+    },
+    [popperRef.current],
```

Comment:
This dependency is not required because `popperRef` will be the same object across multiple renders

https://overreacted.io/how-are-function-components-different-from-classes/

Reply:
addressed, thanks